core/arch/aarch64/emit_utils.c

```diff
@@ -690,8 +690,11 @@ emit_indirect_branch_lookup(dcontext_t *dc, generated_code_t *code, byte *pc,
                                                TLS_MASK_SLOT(ibl_code->branch_type),
                                                OPSZ_16)));
     /* and x1, x1, x2 */
-    APP(&ilist, XINST_CREATE_and(dc, opnd_create_reg(DR_REG_X1),
-                                 opnd_create_reg(DR_REG_X2)));
+    APP(&ilist, INSTR_CREATE_and_shift(dc, opnd_create_reg(DR_REG_X1),
```

Comment:
Update: looks like this was done for _add and others, it's just not there for _and.